src/policy/immortalspace.rs

```diff
@@ -46,12 +44,20 @@ impl<VM: VMBinding> SFT for ImmortalSpace<VM> {
         true
     }
     fn initialize_object_metadata(&self, object: ObjectReference, _alloc: bool) {
-        let old_value = gc_byte::read_gc_byte::<VM>(object);
-        let mut new_value = (old_value & GC_MARK_BIT_MASK) | self.mark_state;
-        if self.header_byte.needs_unlogged_bit {
-            new_value |= self.header_byte.unlogged_bit;
```

Comment:
It was unused (condition was always false).
And it can't be transferred to the refactored code as it is (we don't have header_byte anymore).
So, I think it's better to re-implement it when we actually need it

Reply:
See the comment for the other conversation. 'It is not used' is not a good reason to remove it, as we haven't ported all the plans. But the important question is, is it easy to implement this with the refactored metadata?

Reply:
I don't see a reason to include unused code.
Our porting process should be a bit smarter, otherwise we will be left with a code-base that is not clean and understandable.